SIPS/sip-20.md

```diff
@@ -111,6 +112,35 @@ The caveats MUST NOT have duplicate objects with the same `url` properties.
 
 The `url` MUST start with `wss://` which is a protocol indicator for secure WebSocket connection.
 
+### RPC Methods
+
+This SIP introduces new RPC method for sending data to the external connections.
+
+#### snap_sendData
+
+Snap can use `snap_sendData` RPC method to send data specified within request params.
+The proposed RPC method `snap_sendData` SHOULD be a restricted RPC method requiring user consent before usage via the permission system. The RPC method SHOULD only be available to Snaps.
```

Comment:
I think we should just use `endowment:external-data` for this RPC method as well.